### PR TITLE
Add a translation into Russian for updates/2018/12/class-fields

### PR DESCRIPTION
**Pull request by cerberus-ab**
What's changed, or what was fixed?
- added a translation into Russian (ru) for en/updates/2018/12/class-fields
- for that created folder ru/updates/2018/**12**

I've been talking to @mathiasbynens about this and that he pointed me to WebFundamentals.

**Target Live Date:** does not matter

- [x] This has been reviewed and approved by (NAME)
-- Before contributing I posted the translation on a Russian blog where it has been reviewed and commented.
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
-- I am not sure that I should do that in the case.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by cerberus-ab** [comment]
I signed it!

**Comment by mathiasbynens** [comment]
Thanks for contributing this translation, @cerberus-ab :) I’m glad you reached out on Twitter resulting in this PR!

I’ve asked a Russian-speaking colleague to take a look.

File: src/content/ru/updates/2018/12/class-fields.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    return this._count;
+  }
+  increment() {
+    this._count++;
+  }
+}
+```
+
+Класс предоставляет геттер `value` и метод для инкремента значения в прототипе. Более любопытно, что класс имеет
+конструктор, который инициирует свойство `_count` и выставляет его начальное значение в `0`. Сейчас мы используем
```
Comment (isheludko) [inline src/content/ru/updates/2018/12/class-fields.md]: Probably 
  Обратите внимание, что у класса есть конструктор, который добавляет свойство `_count` и устанавливает его начальное значение в `0`.
would sound a bit better.

File: src/content/ru/updates/2018/12/class-fields.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    console.log('Meow!');
+  }
+}
+```
+
+## Итого
+
+Публичные поля классов доступны, начиная с V8 v7.2 и Chrome 72. Скоро планируется релиз и приватных полей классов.
+
+Вопросы о новых возможностях языка? Комментарии по статье? Не стесняйтесь писать мне в Твиттер [@mathias](https://twitter.com/mathias)!
```
Comment (isheludko) [inline src/content/ru/updates/2018/12/class-fields.md]: У вас есть вопросы о новых возможностях языка? Комментарии к этой статье? ...

File: src/content/ru/updates/2018/12/class-fields.md (hunk 1 above)
Comment (cerberus-ab) [reply src/content/ru/updates/2018/12/class-fields.md]: Yes, it sounds better.

File: src/content/ru/updates/2018/12/class-fields.md (hunk 2 above)
Comment (cerberus-ab) [reply src/content/ru/updates/2018/12/class-fields.md]: Done

**Comment by mathiasbynens** [comment]
This should be live here: https://developers.google.com/web/updates/2018/12/class-fields?hl=ru

@cerberus-ab If you want, you can submit a follow-up PR following the steps here: https://developers.google.com/web/resources/translations#credit This way, you get listed as a contributor!

File: src/content/ru/updates/2018/12/class-fields.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Several proposals expand the existing JavaScript class syntax with new functionality. This article explains the new public class fields syntax in V8 v7.2 and Chrome 72, as well as the upcoming private class fields syntax.
+
+{% include "web/_shared/machine-translation-start.html" %}
```
Comment (mathiasbynens) [inline src/content/ru/updates/2018/12/class-fields.md]: We can remove this, as it's not a machine translation :)

Comment (cerberus-ab) [reply src/content/ru/updates/2018/12/class-fields.md]: Oh, good! I'll append my credit later :) Thanks for that opportunity!

Comment (mathiasbynens) [reply src/content/ru/updates/2018/12/class-fields.md]: Or was this a machine translation with just a few tweaks? In that case, we should keep the line. @PaulKinlan

Comment (cerberus-ab) [reply src/content/ru/updates/2018/12/class-fields.md]: No, this is my own interpretation of the material :)

**Pull request merged**